### DISCUSSION TITLE
Maximum retries exceeded

### DISCUSSION THREAD

**Discussion by Guedesou** [anuraghazra/github-readme-stats#1752] [Q&A] [answered]
Someone saw this problem? I noticed this in several repositories. Someone know how to fix?
![Capturar](https://user-images.githubusercontent.com/56264781/168880891-832ec705-5858-4c86-873e-c3401d513442.PNG)
?

**Comment by lucxsmesquita** [comment]
ward

**Comment by Guedesou** [comment]
Someone passing for this until now?

**Comment by rickstaa** [comment, marked as answer]
This is a known problem that is affecting all users of the public Vercel instance (see #1763 and #1471). For people who experience this problem, please wait until we fix this issue upstream or as mentioned in https://github.com/anuraghazra/github-readme-stats/issues/1471 deploy your own Vercel instance.
